tests/executor/test_construct_model.py

```diff
@@ -98,7 +98,7 @@ def make_local_settings(given_settings):
     j = list(c.cascade_jobs)[1:]
     job_choice = choices.choice(list(range(len(j))), name="job_idx")
     job_kind, job_args = c.cascade_job(j[job_choice])
-    assert job_kind == "estimate_location"
+    assert job_kind.startswith("estimate_location:")
```

Comment:
This feels like an encoding that should be abstracted.

Reply:
Or done away with. It's there mostly because I wanted to keep my changes fairly minimal. I'm not sure there will even be different classes of jobs other than "estimate_location".